docs/guide/deployment.md

```diff
@@ -222,35 +224,57 @@ npm install -g now
     ```json
     {
       "name": "my-example-app",
-      "type": "static",
-      "static": {
-        "public": "dist",
-        "rewrites": [
-          {
-            "source": "**",
-            "destination": "/index.html"
-          }
-        ]
-      },
-      "alias": "vue-example",
-      "files": [
-        "dist"
-      ]
+      "version": 2,
+      "builds": [
+        { "src": "dist/**", "use": "@now/static" }
+      ],
+      "routes": [
+        { "src": "/(.*)", "dest": "dist/$1" }
+      ],
+      "alias": "vue-example"
     }
     ```
-
-    You can further customize the static serving behavior by consulting [Now's documentation](https://zeit.co/docs/deployment-types/static).
+    
+   In case you want to deploy an application with router mode set to history, the config file should look like the following (if you have different folder names, update your config accordingly):
```

Comment:
```suggestion
   In case if you want to deploy an application with router mode set to `history`, the config file should look like the following (if you have different folder names, update your config accordingly):
```